docs/rfcs/002-archival_storage/README.md

```diff
@@ -555,28 +555,28 @@ Response:
       }
     ]
   },
-  "locations": [
-    {
-      "type": "Location",
-      "provider": {
-        "type": "Provider",
-        "id": "aws-s3-ia",
-        "label": "AWS S3 - Infrequent Access"
-      },
-      "bucket": "bucketname-access",
-      "path": "digitised/b24923333",
-      "url": "http://bucketname-access.s3-eu-west-1.amazonaws.com/digitised/b24923333"
+  "primaryLocation": {
```

Comment:
Any opinions on whether this should be `location` or `primaryLocation`?

Reply:
I would say `location`. The name of the bucket is `wc-storage`, not `wc-storage-primary`.

Reply:
I think I also prefer `location` for the same reason.

Reply:
Ok will change 👍